packages/webapp/webapp_server.js

```diff
@@ -1,5 +1,6 @@
 import assert from "assert";
 import { readFileSync } from "fs";
+import { chmodSync } from "fs";
```

Comment:
import `readFileSync` and `chmodSync` in the same line?

Reply:
Thanks, I will make that change.